packages/ui-styles/scss/lib/_mixins.scss

```diff
@@ -5,3 +5,9 @@
   -webkit-border-radius: $radius;
   -moz-border-radius: $radius;
 }
+
+@mixin button-border-radius($radius: $button-border-radius) {
```

Comment:
@frankjlin16 great idea! Can you make this more generic and just call it `border-radius`?